### PR TITLE
Add details of CUDAEnsemble standby blocking.

### PR DESCRIPTION
**Pull request by Robadob**
See: https://github.com/FLAMEGPU/FLAMEGPU2/pull/930

### CODE REVIEW

**Review by ptheywood — APPROVED** [review]
Looks fine to merge once the main PR is merged.

**Review by mondus — APPROVED** [review]

**Pull request merged**
